chart/values.yaml

```
@@ -27,6 +27,15 @@ service:
   type: ClusterIP
 mongodb:
   replicas: 1
+watcher:
+  emailPolicyInformer:
+    selector:
```

This should be an object, shouldn't it?

```suggestion
    selector: {}
```